baip-0002.md

```diff
@@ -1,9 +1,9 @@
     BAIP: 0002
     Title: Reform of loopholes in feed price mechanism
     Authors: cn-vote  bitshareschina@163.com
-    Status: Draft
+    Status: Accepted
```

Comment:
Disagree. Many pull request review comments await responses: https://github.com/bitshares/baips/pull/9

Reply:
> Disagree. Many pull request review comments await responses: #9

Active, any questions here can be answered

Reply:
That's not true, the majority of the BAIP2 file is not available to be commented on in the 'files changed' section. Answer the questions in #9 than dodge them - if they hold no validity then surely it'd be simple to respond to them?

Reply:
I think the content of BAIP002 is very complete.
 In fact, this plan has been discussed in the forum for more than a month. 
We should not stay in endless discussions without doing anything of practical significance.

Reply:
> I think the content of BAIP002 is very complete.

I disagree, at the very least there are multiple grammar errors.

>  We should not stay in endless discussions without doing anything of practical significance.

Then address comments in the pull request review process than disregard BAIP processes. Being unable to directly defend ones own work is concerning & indicative of poor quality.

Reply:
@grctest would you like to create a new pull request to correct the grammar errors so we can go forward? I guess it's too hard for the original author(s) since his/their English skill is not as good as yours.

Reply:
Given the direct threat of legal action by the author, no. 

My original pull request comments are still there if anyone's interested.